Xamarin.Forms.Core/NativeBindingHelpers.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
+{
+	static class NativeBindingHelpers
+	{
+		public static void SetBinding<TNativeView>(TNativeView target, string targetProperty, BindingBase bindingBase, string updateSourceEventName) where TNativeView : class
+		{
+			if (string.IsNullOrEmpty(updateSourceEventName))
+				throw new ArgumentNullException(nameof(updateSourceEventName));
+
+			var eventWrapper = new EventWrapper(target, targetProperty, updateSourceEventName);
```

Comment:
https://github.com/xamarin/Xamarin.Forms/pull/278/files#diff-0faacf4d695182912f1ba727d3b6a67bR362 this tests for the non existing event , need test for te property